src/Files.App/Utils/RecentItem/RecentItems.cs

```diff
@@ -43,9 +43,15 @@ public sealed class RecentItems : IDisposable
 			}
 		}
 
-		public RecentItems()
+		public IUserSettingsService UserSettingsService { get; }
```

Comment:
```suggestion
		private readonly IUserSettingsService UserSettingsService;
```
I don't think this needs to be public.